todo_app/data/trello_items.py

```diff
@@ -0,0 +1,84 @@
+# This code uses the 'requests' library to call trello API
+# http://docs.python-requests.org
+import requests
+import os
+
+list_ids = {"to_do":"6501688a993cb949a6478d07",
+            "doing":"6501688bc072b227f990762d",
+            "done":"6501688a1c964b99f5ea6d20"
+}
+
+
+class TrelloManager:
+  
+    board_id = os.environ.get('BOARD_ID')
+    APIKey = os.environ.get('TRELLO_API_KEY')
+    APIToken = os.environ.get('TRELLO_TOKEN')
+    url = f"https://api.trello.com/1/boards/{board_id}/lists"
```

Comment:
This is a nice way to structure it, but be warned it may trip you up in the following exercise when we introduce testing - this is because these static variables will be loaded in when this file is _imported_, which often happens before your tests have time to set up the fake environment variables. If that's the case, one option you could use is to make these functions that get evaluated when you're actually using them, e.g.:
```python
    @staticmethod
    def board_id():
        return os.environ.get('BOARD_ID')
```

And then you can access them with `TrelloManager.board_id()` rather than `TrelloManager.board_id` etc.